phpBB/assets/javascript/converter.js

```diff
@@ -0,0 +1,621 @@
+/**
```

Comment:
Just for testing. As you see it does some console.logs which will help me debug. Final pushes wont have this at all.